distribution/docker/debian/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM debian:bullseye-slim AS builder
 
+WORKDIR /vector
+
 COPY vector_*.deb ./
-RUN dpkg -i vector_*_$(dpkg --print-architecture).deb
+RUN dpkg -i "vector_*_$(dpkg --print-architecture).deb"
```

Comment:
Don't forget these! :trollface:

Reply:
_I should just assign this to you @hhromic_

Reply:
Fixed c78fa130d